analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -319,7 +321,8 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
                 binaryArtifact = binaryRemoteArtifact,
                 sourceArtifact = sourceRemoteArtifact,
                 vcs = vcsFromPackage,
-                vcsProcessed = PackageManager.processPackageVcs(vcsFromPackage)
+                vcsProcessed = localDirectory?.let { PackageManager.processProjectVcs(localDirectory, vcsFromPackage) }
```

Comment:
I believe it would be more common to use `it` instead of a second time `localDirectory` inside `let`.

Reply:
Done.